src/core_plugins/kibana/public/management/react/index-pattern/index-pattern-view/components/index-pattern-fields/index-pattern-fields.container.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { IndexPatternFields as IndexPatternFieldsComponent } from './index-pattern-fields.component';
```

Comment:
How come the files are using `-` for a separator instead of `_`? Is this an intentional change in style?  I thought our convention was underscore.

Reply:
Ah yea. That's my bad. Just ignore the discrepancies :)